admin/sql/updates/20210215-mbs-11268.sql

```diff
@@ -0,0 +1,15 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+CREATE OR REPLACE VIEW medium_track_durations AS
```

Comment:
I removed the `SET search_path` at the beginning of this script (and pushed a commit to remove it from the other scripts). It's already injected at the beginning of the transaction.